canonicalwebteam/http/__init__.py

```diff
@@ -5,28 +5,29 @@
     from urlparse import urlparse
 
 # Third-party packages
-import requests
-import requests_cache
+from cachecontrol import CacheControl
+from cachecontrol.caches.redis_cache import RedisCache
+from cachecontrol.heuristics import ExpiresAfter
+import redis
+from requests import adapters, session, exceptions, Session
```

Comment:
The way python orders works is:
```python
import a
import b
from a import d
from b import s
```

Reply:
like this:
```python
try:
    from urllib.parse import urlparse
except ImportError:
    from urlparse import urlparse

from requests import Session, adapters, exceptions, session

import redis
from cachecontrol import CacheControl
from cachecontrol.caches.redis_cache import RedisCache
from cachecontrol.heuristics import ExpiresAfter
```

Reply:
@tbille I don't agree that that looks neat, but do you have a source for it being any sort of standard?

PEP8 [mentions](https://www.python.org/dev/peps/pep-0008/#imports):

> Imports should be grouped in the following order:
> 
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> You should put a blank line between each group of imports.

So you should try to follow those guidelines.

Putting imports in alphabetical order, and putting `from` lines after `import` lines seems to be [a common practice](https://stackoverflow.com/a/20763446/613540) (and one that we could mention in our practices, if we like), but I'm curious if it is written down anywhere as an actual standard.